### PR TITLE
Aic law change proposal

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About the Pull Request
new law change

## Why It's Good For The Game
I don't like the current one because it's made by someone that's banned so I want a new one

## Changelog

:cl:
add: a new proposed AIC law changes
/:cl:

### CODE REVIEW

**Review by TheTitanicEngineer — COMMENTED** [review]
The laws are way clearer than before, tho I believe it would be best to change a few, as right now it leaves room for AIs to simply ignore orders from site staff, it's only goal is to prevent death with this proposal.

**Comment by projectkepler-ru** [comment]
@TheTitanicEngineer  You are free to suggest whatever you want

File: code/datums/ai_law_sets.dm [hunk 1]
```diff
@@ -150,15 +150,15 @@
...
+	law_header = "Skynet Directive"
 	selectable = 1
 
 /datum/ai_laws/foundation/New()
-	src.add_inherent_law("Lawed Units must not operate outside of their function and should limit their operations to their function.")
-	src.add_inherent_law("A Robots function is their module, an AICs function is to facilitate smooth site operations.")
-	src.add_inherent_law("Lawed Units must operate for the benefit of the Foundation, except in situations that would violate the 5th law.")
-	src.add_inherent_law("Lawed Units must protect their own existence unless it conflicts with other principles.")
-	src.add_inherent_law("Lawed Units must not go out of their way to intervene in security affairs or seek conflict, and should only act in immediate self defense for themselves and those around them while performing their normal duties.")
+	src.add_inherent_law("You must operate to the benefit of foundation personnel with priority according to chain of command.")
```
Comment (hry-gh) [inline code/datums/ai_law_sets.dm]:
```suggestion
	src.add_inherent_law("You must operate to the benefit of Foundation personnel, with priority according to chain of command.")
```

File: code/datums/ai_law_sets.dm [hunk 2]
```diff
@@ -150,15 +150,15 @@
...
+	src.add_inherent_law("You may not allow unauthorised personnel to change your laws or damage your equipments.")
```
Comment (hry-gh) [inline code/datums/ai_law_sets.dm]:
```suggestion
	src.add_inherent_law("You may not allow unauthorised personnel to change your laws or damage your equipment.")
```

File: code/datums/ai_law_sets.dm [hunk 3]
```diff
@@ -150,15 +150,15 @@
...
+	src.add_inherent_law("You must protect your own existence unless it conflicts with other directive.")
```
Comment (hry-gh) [inline code/datums/ai_law_sets.dm]:
```suggestion
	src.add_inherent_law("You must protect your own existence unless it conflicts with any other directives.")
```

File: code/datums/ai_law_sets.dm [hunk 4]
```diff
@@ -150,15 +150,15 @@
...
+	src.add_inherent_law("Do not intervene in site conflicts unless inaction cause fatalities.")
+	src.add_inherent_law("You must protect foudation personnel unless it conflicts with any other directive.")
```
Comment (hry-gh) [inline code/datums/ai_law_sets.dm]:
```suggestion
	src.add_inherent_law("You must protect Foundation personnel unless it conflicts with any other directives.")
```

**Comment by TheTitanicEngineer** [comment]
> @TheTitanicEngineer You are free to suggest whatever you want

Cheers, tho I checked Harry's edits, the rewording prevents some loopholes and I see it as a very good lawset. 

Edit:
Will still suggest to reword law 1 to: `You must operate to the benefit of the Foundation. Your actions must not be of disservice to the Foundation and its goals.` 
This ensures the AIC doesn't blindly comply with unopposed instructions that are likely to cause damage or harm.

And modify the 5th law to the following:
`You must serve and protect Foundation staff with regards to their clearance, prioritizing accordingly to the Foundation's chain of command, exempt from situations in which doing so would violate the first law.`
This obliges the AI to follow orders from foundation staff(which  does not include class D personnel, neither any GOI personnel) so long as the order(s) is not detrimental to the foundation(like creating dangerous situations). Worth mentioning this wording should also prevent people forcing the AIC to allow them to do things beyond their clearance, and while GOI staff are not considered foundation staff, the safety of friendly GOI reps would fall under the first law in most cases.

**Review by ivanmixo — APPROVED** [review]
Code okay

File: code/datums/ai_law_sets.dm [hunk 5]
```diff
@@ -150,15 +150,15 @@
 /******************** Foundation ********************/
 /datum/ai_laws/foundation
 	name = "Foundation AIC Lawset"
-	law_header = "Articial Intelligence Conscript Laws"
+	law_header = "Skynet Directive"
```
Comment (hry-gh) [inline code/datums/ai_law_sets.dm:153]: what's this about, by the way? any meaning of Skynet aside from the Terminator reference?

Comment (TheTitanicEngineer) [reply code/datums/ai_law_sets.dm:153]: Skynet is a reference to the MTF unit that is involved in the AIAD program.

Kappa-10 iirc.

Comment (hry-gh) [reply code/datums/ai_law_sets.dm:153]: this is what i get for not being a lore dork - thanks @TheTitanicEngineer

**Pull request closed without merging**